docs/resource-manager-client.rst

```diff
@@ -0,0 +1,18 @@
+.. toctree::
+  :maxdepth: 0
+  :hidden:
+
+Client
+~~~~~~
+
+.. automodule:: gcloud.resource_manager.client
+  :members:
+  :undoc-members:
+  :show-inheritance:
+
+Connection
```

Comment:
Is there... any way to have a header here, but not have it show up in the left nav? @dhermes That is -- I want "Client" in the left nav, but not "Connection"